libraries/AP_HAL_ChibiOS/hwdef/AeroFox-Airspeed/hwdef.dat

```diff
@@ -0,0 +1,24 @@
+include ../f103-periph/hwdef.inc
+
+define CAN_APP_NODE_NAME "org.ardupilot.AeroFox_Airspeed"
+
+
+undef  HAL_COMPASS_MAX_SENSORS 1
+define HAL_COMPASS_MAX_SENSORS 2
```

Comment:
why set MAX_SENSORS to 2? We only support one compass on an AP_Periph node so far. Note that you can have MAX_SENSORS==1 and still probe for more than one compass type